### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 106)

### LINKED ISSUE #3736: [devtools-translate] Portuguese - What's New in DevTools (Chrome 106)
Post: https://developer.chrome.com/blog/new-in-devtools-106/
Translated by: Sep 23

Translator: @khaosdoctor
Reviewers: @alvarocamillont

### PR DESCRIPTION
**Pull request by khaosdoctor**
Fixes #3736

Changes proposed in this pull request:

- Translate the files to Portuguese

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks @khaosdoctor over to @alvarocamillont for review!

File: site/pt/blog/new-in-devtools-106/index.md [hunk 1]
```diff
@@ -57,124 +65,155 @@ async increment() {
...
 
-Chromium bug: [1334585](https://crbug.com1334585)
+Issue do Chromium: [1334585](https://crbug.com1334585)
 
 
 <!-- ### Automatically ignore known third-party scripts {: #auto-ignore } -->
+### Ignorar os scripts de terceiros conhecidos automaticamente {: #auto-ignore }
 
 <!-- Identify issues in your code quicker during debugging because DevTools now automatically adds known third-party scripts to the ignore list. -->
+identifique problemas no seu código mais rapidamente durante o debug porque o DevTools agora adiciona automaticamente todos os scripts de terceiros que são conhecidos à sua lista de ignorados.
```
Comment (alvarocamillont) [inline site/pt/blog/new-in-devtools-106/index.md:90]: Identifique

**Review by alvarocamillont — COMMENTED** [review]
some minor adjustments

File: site/en/blog/new-in-devtools-106/index.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ Chromium bug: [1336604](https://crbug.com/1336604)
 
 Use the new **Interactions** track in the **Performance** panel to visualize interactions and track down potential responsiveness issues. 
 
-For example, [start a performance recording](/docs/devtools/evaluate-performance/#record ) on this [demo page](https://coffee-cart.netlify.app/?ad=1). Click on a coffee and stop recording. Two interactions show in the **Interactions** track. Both interactions have the same IDs, indicating the interactions are triggered from the same user interaction.
+For example, [start a performance recording](/docs/devtools/evaluate-performance/#record) on this [demo page](https://coffee-cart.netlify.app/?ad=1). Click on a coffee and stop recording. Two interactions show in the **Interactions** track. Both interactions have the same IDs, indicating the interactions are triggered from the same user interaction.
```
[the remark this answers is not in the document]
Comment (khaosdoctor) [reply site/en/blog/new-in-devtools-106/index.md:131]: I made it, there were some weird spacing

File: site/pt/blog/new-in-devtools-106/index.md (hunk 1 above)
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-106/index.md:90]:
```suggestion
Identifique problemas no seu código mais rapidamente durante o debug porque o DevTools agora adiciona automaticamente todos os scripts de terceiros que são conhecidos à sua lista de ignorados.
```

**Comment by alvarocamillont** [comment]
Ok! LGTM

**Review by jecfish — APPROVED** [review]

File: site/pt/blog/new-in-devtools-106/index.md [hunk 3]
```diff
@@ -3,19 +3,20 @@ layout: "layouts/blog-post.njk"
 title: "O que há de novo no DevTools (Chrome 106)"
 authors:
   - jecelynyeen
+  - khaosdoctor
```
Comment (jecfish) [inline site/pt/blog/new-in-devtools-106/index.md:6]:
```suggestion
```

File: site/pt/blog/new-in-devtools-106/index.md [hunk 4]
```diff
@@ -3,19 +3,20 @@ layout: "layouts/blog-post.njk"
...
+  - alvarocamillont
```
Comment (jecfish) [inline site/pt/blog/new-in-devtools-106/index.md:7]:
```suggestion
```

**Comment by jecfish** [comment]
Thank you both!

**Pull request merged**